src/Resources/config/doctrine_mongodb.xml

```diff
@@ -17,7 +17,7 @@
             <tag name="sonata.admin.guesser.doctrine_mongodb_list"/>
         </service>
         <service id="sonata.admin.guesser.doctrine_mongodb_list_filter" class="Sonata\DoctrineMongoDBAdminBundle\Guesser\FilterTypeGuesser">
-            <tag name="sonata.admin.guesser.doctrine_mongodb_list"/>
+            <deprecated>The "%service_id%" service is deprecated since sonata-project/doctrine-mongodb-admin-bundle 3.x and will be removed in 4.0.</deprecated>
```

Comment:
```suggestion
            <deprecated>The "%service_id%" service is deprecated since sonata-project/doctrine-mongodb-admin-bundle 3.x and will be removed in 4.0. Use "sonata.admin.guesser.doctrine_mongodb_datagrid" service instead.</deprecated>
```